highway will fail pkgconf test pipeline

If the pkgconf test pipeline is added to highway it will fail in the following way:

```
2024/11/12 23:11:18 INFO running step "test/pkgconf"
2024/11/12 23:11:18 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:11:18 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:11:18 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:11:18 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:11:18 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + basename /home/build/melange-out/highway-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + dev_pkg=highway-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + apk info -L highway-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + grep -q ^Name: usr/lib/pkgconfig/libhwy-contrib.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + basename usr/lib/pkgconfig/libhwy-contrib.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + lib_name=libhwy-contrib uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + echo usr/lib/pkgconfig/libhwy-contrib.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + grep -q '^usr/lib/pkgconfig/libhwy-contrib.pc$\|^usr/share/pkgconfig/libhwy-contrib.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + pkgconf --exists libhwy-contrib uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + grep -q ^Version: usr/lib/pkgconfig/libhwy-contrib.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + pkgconf --modversion libhwy-contrib uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 INFO 1.2.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + grep -q ^Libs: usr/lib/pkgconfig/libhwy-contrib.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + pkgconf --libs libhwy-contrib uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 INFO -lhwy_contrib uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + grep -q ^Cflags: usr/lib/pkgconfig/libhwy-contrib.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + pkgconf --cflags libhwy-contrib uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 INFO uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + grep -q ^Name: usr/lib/pkgconfig/libhwy-test.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + basename usr/lib/pkgconfig/libhwy-test.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + lib_name=libhwy-test uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + echo usr/lib/pkgconfig/libhwy-test.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + grep -q '^usr/lib/pkgconfig/libhwy-test.pc$\|^usr/share/pkgconfig/libhwy-test.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + pkgconf --exists libhwy-test uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN + pkgconf --print-errors --exists libhwy-test uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN Package gtest was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN Perhaps you should add the directory containing `gtest.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 WARN Package 'gtest', required by 'libhwy-test', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:18 INFO pod ef1795539aebf4a6b6549498f97239cf7411be02e1e87d5e268830dc64eb5d46 terminated
2024/11/12 23:11:18 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:11:18 INFO   workspace dir: /home/user/tmp/melange-workspace-3109005586
2024/11/12 23:11:18 INFO   guest dir: /home/user/tmp/melange-guest-2680819198
2024/11/12 23:11:18 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/highway] Error 1
```

--- a/highway.yaml
+++ b/highway.yaml
@@ -1,7 +1,7 @@
 package:
   name: highway
   version: 1.2.0
-  epoch: 1
+  epoch: 2
   description: Performance-portable, length-agnostic SIMD with runtime dispatch
   copyright:
     - license: Apache-2.0
@@ -60,6 +60,9 @@ subpackages:
     pipeline:
       - uses: split/dev
     description: highway dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   enabled: true

